### DISCUSSION TITLE
New lgMobileOnly directive

### DISCUSSION THREAD

**Discussion by NemirStephan** [Legal-and-General/canopy#279] [Ideas] [answered]
I would like to propose a new LgMobileOnly directive that can be applied to html that only needs to be displayed in mobile view.

**Comment by elenagarrone** [comment, marked as answer]
Sounds like a nice to have 👍  but I'm wondering if it should be like `LgHideAtBreakpoint` or something similar so that you can affect any breakpoint instead of just mobile.

**Reply by NemirStephan** [reply]
That would make more sense to me

**Reply by owensgit** [reply]
Yes I think more generic approach would be great. Although I would advocate `ShowAtBreakpoint` or `ShowAt`, as then it's a bit more mobile-first centric - e.g we show something at the medium breakpoint, meaning it's hidden by default all devices up to that breakpoint.

E.g.
`<lg-card showAtBreakpoint="md">`

Not sure if I prefer `showAtBreakpoint` or just simply `showAt` 🤔 

This idea was also mentioned on a recent PR:
https://github.com/Legal-and-General/canopy/pull/167#issuecomment-762920983

**Reply by owensgit** [reply]
PR opened with `showAt` and `hideAt` directives: https://github.com/Legal-and-General/canopy/pull/305
